public/js/actions/breakpoints.js

```diff
@@ -37,10 +53,11 @@ function addBreakpoint(location, { condition, getTextForLine } = {}) {
       breakpoint: bp.toJS(),
       condition: condition,
       [PROMISE]: Task.spawn(function* () {
-        const { id, actualLocation } = yield client.setBreakpoint(
+        const res: any = yield client.setBreakpoint(
```

Comment:
Destructuring is supported, I just ran into an issue here and I can't remember what it was. I didn't spend much time on this part of the file, just had to get something working.